### PR TITLE
Add scripts/md5sums.sh

### PR DESCRIPTION
**Pull request by manics**
The script I've been using for calculating MD5 checksums.
- Sorts all files before calculating the checksums
- Outputs to stdout

### CODE REVIEW

**Comment by dominikl** [comment]
That's handy, thanks. Works fine 👍

**Comment by sbesson** [comment]
:+1: for adding a generic script for computing checksums that anyone in the team can run.

I thought we had moved away from MD5  in favor of SHA a few years ago primarily because of the known collision issues in MD5s. For published studies, collisions might be less of an issue. However, would there a drawback other than increasing computation time e.g. to using SHA-1?

**Comment by manics** [comment]
I've changed this so the checksum algorithm is specified as the first parameter

**Comment by jburel** [comment]
Could the file be renamed to match the fact that it is not only md5?, e.g. ``checksums_calculator.sh``

**Comment by jburel** [comment]
I have deleted my comment, the file has been renamed

**Comment by jburel** [comment]
running for example 
* ``bash checksums.sh md5sum <directory>`` does not work, it always returns ``Invalid checksum command: md5sum``

Also  I will be good to have a default e.g. sha1sum

File: scripts/checksums.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+alg="$1"
+target="$2"
+
+if ! command -v "$alg"; then
+    echo "Invalid checksum command: $alg"
+    exit 2
+fi
+
+cd "$target"
+find . -type f -print0 | sort -z | xargs -0 -n1 "$alg"
```
Comment (sbesson) [inline scripts/checksums.sh:26]:
```suggestion
find * -type f -print0 | sort -z | xargs -0 -n1 "$alg"
```

Should remove the leading `./` from the generated file list

**Review by sbesson — APPROVED** [review]

**Comment by jburel** [comment]
In a conda environment with coreutils installed.
```
bash scripts/checksums.sh sha256sum test
/Applications/anaconda2/envs/omero_and_web/bin/sha256sum
48dd5f31dbc7058f75739f45cc81e6490eae7aaa92a242dd652b58e02b90b17c  all_tests.py
a8f2f7e730d9e70090c470e82659331e6e95ada24831ed5e8f8719d3c004d813  test_file_pattern.py
065cc2eeab82cae8e298f5f1079913d08279cdb31981676d4633b37b00ac74e2  test_screenio.py
```
and
```
bash scripts/checksums.sh sha256sum1 test
Invalid checksum command: sha256sum1
````

If someone wants to run the script in a terminal on a mac. This will not work by default, we may want to add a note

**Review by jburel — APPROVED** [review]

**Pull request merged**